### DISCUSSION TITLE
今天把5片PCB都打样了,感谢作者！

### DISCUSSION THREAD

**Discussion by eqio** [peng-zhihui/ElectronBot#36] [General]
数量是5件，有部分如果有网友需要，可以赠送。🙂
玩的朋友可以一起交流下，
感谢作者的开源，让我们能学到这么多。

**Comment by Liu-Mage** [comment]
你好，请问怎么联系您呢？

**Reply by eqio** [reply]
18148412989

**Reply by anderswang** [reply]
建个什么群吧。

**Reply by flycat1212** [reply]
拉个群吧

**Reply by Liu-Mage** [reply]
那各位大佬能否建个QQ群？

**Comment by eqio** [comment]
大家可以直接去嘉立创大板，不SMT的话，5片一共才100元，比较便宜的。
如果一定要撸嘉立创的羊毛，可以把文件打开后丝印改动一下（作者名字留着^_^）再去做板，一天有2个免费的订单。
直接按文件加工会认为相同文件，有拆板嫌疑。20元一个规格。


另外，方便大家讨论，拉了一个群。不知道稚晖君大佬介意不！

![wei](https://user-images.githubusercontent.com/76235694/158196471-0a5ce5c7-60ad-4305-9ef6-cabae3f9e20e.jpg)

**Reply by w1453746894** [reply]
群号过期了
